training/distributed_training/pytorch/model_parallel/vit/requirements.txt

```diff
@@ -0,0 +1,10 @@
+transformers==v4.21.0
```

Comment:
Do you need all these packages?

Reply:
New checkpoint API changes are in transformers v4.21.0.

Reply:
I mean all the packages